content/php-client/resources/PAM/asset-variation-files.md

```diff
@@ -60,6 +60,13 @@ $product = $client->getAssetVariationFileApi()->downloadFromLocalizableAsset('ch
 file_put_contents('/tmp/chicagoskyline-mobile.jpg', $product->getContents());
 ```
 
+From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the Mime type from the response.
```

Comment:
MIME
```suggestion
From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the MIME type from the response.
```